doc/whats-new.rst

```diff
@@ -276,7 +280,6 @@ New Features
 - Expose ``use_cftime`` option in :py:func:`~xarray.open_zarr` (:issue:`2886`, :pull:`3229`)
   By `Samnan Rahee <https://github.com/Geektrovert>`_ and `Anderson Banihirwe <https://github.com/andersy005>`_.
 
-
```

Comment:
I didn't mean to delete the extra newline here.  Should I put it back in?
```suggestion

```

Reply:
It's OK